author/backends.py

```diff
@@ -39,14 +43,21 @@ class AuthorDefaultBackend(object):
 
     def __init__(self):
         required_middleware = 'author.middlewares.AuthorDefaultBackendMiddleware'
-        if required_middleware not in settings.MIDDLEWARE_CLASSES and required_middleware not in settings.MIDDLEWARE:
-            raise ImproperlyConfigured(
+        new_required_middleware = 'author.middlewares.AuthorDefaultBackendMiddlewareNewStyle'
+        if django.VERSION > (1, 10):
```

Comment:
The version check is a bit unintuitive for me. I would prefer the condition to be `django.VERSION >= (1, 10)` since both conditions match all Django 1.10 subversions. Unless there is a reason for this.
Please change this also in `author/middlewares.py`.